installation/README.md

```diff
@@ -0,0 +1,92 @@
+# Installing Config Sync manually
+
+This document is a guide on how to manually install Config Sync on a Kubernetes
+cluster without Fleet Management.
+
+The recommended approach is to [install Config Sync managed by fleet](https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/installing-config-sync).
+
+This document includes configurations for some common user journeys, but is not
+an exhaustive list of how a Config Sync installation can be customized.
+
+## Pre-requisites
+
+This guide requires the following command line tools:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+
+## Uninstall Config Management
+
+If you have previously installed Config Management on your cluster, Config Management
+must first be uninstalled before following this guide to install/upgrade Config Sync.
+
+You can check if Config Management is installed on your cluster with the following command:
+
+```shell
+kubectl get configmanagement
+```
+
+If the output is not empty, Config Management is currently installed on the cluster
+and must be uninstalled before proceeding. To uninstall Config Management, there are
+two options:
+- Use `nomos migrate --remove-configmanagement`
+- Use the [uninstall script](uninstall_configmanagement.sh) provided in this directory.
+
+**Prerequisites**:
```

Comment:
Suggest moving the prerequisites to the beginning of the uninstall section.

Reply:
done